app/main.py

```diff
@@ -1,8 +1,26 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.__class__.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    married_dict = {}
+    result = []
+
+    for item in people:
+        person = Person(name=item["name"], age=item["age"])
+        married_dict[item["name"]] = person
+        result.append(person)
+
+    for item in people:
+        person_obj = married_dict[item["name"]]
```

Comment:
You can create a variable for 'person' and then use it, instead of adding it to the dictionary